src/main/java/frc/utils/time/TimeUtils.java

```diff
@@ -0,0 +1,48 @@
+package frc.utils.time;
+
+import edu.wpi.first.hal.HALUtil;
+import frc.utils.Conversions;
+import frc.utils.alerts.Alert;
+import frc.utils.alerts.AlertManager;
+import frc.utils.alerts.PeriodicAlert;
+import org.littletonrobotics.junction.Logger;
+
+public class TimeUtils {
+
+	public static final double DEFAULT_CYCLE_TIME_SECONDS = 0.02;
```

Comment:
why not in TimeConstants?

Reply:
Constants that are useful out of this class should be in the open class.
If I want something about the time I always will go to time utils and no need to think time utils or time constants.
Constants are just for inner use without being exposed outside